lib/node_modules/@stdlib/math/base/ops/cneg/examples/index.js

```diff
@@ -19,23 +19,21 @@
 'use strict';
 
 var Complex128 = require( '@stdlib/complex/float64' );
-var randu = require( '@stdlib/random/base/randu' );
-var real = require( '@stdlib/complex/real' );
-var imag = require( '@stdlib/complex/imag' );
-var cneg = require( './../lib' );
+var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
+var cneg = require( '@stdlib/math/base/ops/cneg' );
```

Comment:
```suggestion
var cneg = require( './../lib' );
```

We use relative paths in `examples/index.js` files.